<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>batch_manage</class>
 <widget class="QWidget" name="batch_manage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(225, 255, 255)</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>601</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;
background-color:rgb(240, 255, 255);
color:rgb(255, 0, 0);
gridline-color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string>产品批次</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
COLOR:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>产品类型</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
backgroundcolor:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
COLOR:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>批次</string>
    </property>
    <property name="margin">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;
backgroundcolor:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
COLOR:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>产品尺寸</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
COLOR:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>日期</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
COLOR:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>创建人</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
backgroundcolor:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
backgroundcolor:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
backgroundcolor:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
    </property>
    <item>
     <property name="text">
      <string>直角屏</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>R角水滴</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>320</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
backgroundcolor:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>320</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
backgroundcolor:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
backgroundcolor:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
    </property>
    <item>
     <property name="text">
      <string>无屏蔽</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>屏蔽屏蔽左侧</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>屏蔽屏幕右侧</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;
backgroundcolor:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
COLOR:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>屏蔽区域</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
COLOR:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>屏幕型号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
COLOR:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>分辨率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
COLOR:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>刷屏方式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
COLOR:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>外形尺寸</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
COLOR:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>配置方案</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
backgroundcolor:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
    </property>
    <item>
     <property name="text">
      <string>720*1280</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
backgroundcolor:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
    </property>
    <item>
     <property name="text">
      <string>正扫</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
backgroundcolor:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
    </property>
    <item>
     <property name="text">
      <string>L*w*h</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
backgroundcolor:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
    </property>
    <item>
     <property name="text">
      <string>玻璃COG</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
